x-pack/legacy/plugins/siem/public/components/page/hosts/alerts_table/translations.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const ALERTS = i18n.translate('xpack.siem.alertsTable.hostsTitle', {
```

Comment:
Looks like most of these aren't being used. Did you want to use them as Column Name aliases? If so, you can add them where the `columnHeader[]`'s are defined. Otherwise I would remove them.

Reply:
Yes, those should have been removed!